### DISCUSSION TITLE
Could not load file or assembly

### DISCUSSION THREAD

**Discussion by sulan-blob** [jbevain/cecil#750] [Q&A]
Hey,

I get a strange error "Error: Could not load signature of DG.DOTweenEditor.ASMDEFManager:ValidateProASMDEFReferences due to: Could not load file or assembly 'DOTween, Version=1.0.0.0", even after I give the DefaultAssemblyResolver.AddSearchDirectory(..) the full path to resolve DOTween.dll. But every other *.dll is resolved properly, what could be the problem?

T & R,
Sulan

**Comment by jbevain** [comment]
Hi,

Unfortunately I have no way of knowing.

The most reasonable explanation would be that the assembly is not in a path that resolver is looking at, or that it's not in a form that Cecil understands.

You'll likely need to debug what's happening.
